scripts/irods/test/test_iticket.py

```diff
@@ -874,3 +874,110 @@ def get_modification_and_creation_time():
         previous_ts = modify_ts
         modify_ts, _ = get_modification_and_creation_time()
         self.assertGreater(modify_ts, previous_ts)
+
+    def test_tickets_update_catalog_modify_time_when_modified_user__issue_7125(self):
+
+        def get_modification_and_creation_time():
+            out, err, ec = self.admin.run_icommand(['iquest', '%s...%s', "select TICKET_MODIFY_TIME, TICKET_CREATE_TIME where TICKET_STRING = '{}'".format(ticket_string)])
+            self.assertEqual(ec, 0)
+            self.assertEqual(len(err), 0)
+
+            out = out.strip().split('...')
+            modification_time = out[0]
+            creation_time = out[1]
+
+            return int(modification_time), int(creation_time)
+
+        ticket_string = "issue_7125_ticket_user"
+        file_name = "issue_7125_file_user"
+        self.user.assert_icommand(['itouch', file_name])
+
+        self.user.assert_icommand(['iticket', 'create', 'write', file_name, ticket_string])
+
+        modify_ts, create_ts = get_modification_and_creation_time()
```

Comment:
Does anything use `create_ts`?

Reply:
`create_ts` is only used once. Initially I was going to compare the modify time to the create time for every modification but realized that it would be redundant after the first time the modification time was different.

Reply:
After creating the ticket, are the mtime and ctime identical?

Reply:
Immediately after creating the ticket, the mtime and ctime will be identical. After a ticket is modified, the ctime and mtime should differ.

Reply:
It seems you don't need the ctime for this test. You agree?

Reply:
Yeah that is true. I will update the test.

Reply:
Thinking on this a little, it might be worth doing the comparison. It's only redundant if we assume that every operation properly updates the mtime and also *does not* update the create time. Thoughts?

Reply:
I think that makes sense. I can check that the creation time is equal to the original create time and not equal to the modification time.